site/content/docs/include/observability.en.md

```diff
@@ -0,0 +1,7 @@
+<div class="separator"></div>
+
+<a id="observability" href="#observability" class="field">`observability`</a> <span class="type">Map</span>      
+The `observability` section lets you configure ways to measure your service's current state. Currently, only tracing configuration is supported.
+
+<span class="parent-field">observability.</span><a id="observability-tracing" href="#observability-tracing" class="field">`tracing`</a> <span class="type">String</span>    
+The vendor to use for tracing. Currently, only `awsxray` is supported.
```

Comment:
I think we need some documentation on how to use `otel` too so that people can leverage this functionality, what do you think?

Reply:
I was thinking the same thing, we don't offer much in the way of "this is how to do it." I was wondering if a more detailed doc under the `Developing` sidebar would make sense - thoughts? Or just something short here?

Reply:
That's a great idea!

And we can link to that advanced guide in Developing from here